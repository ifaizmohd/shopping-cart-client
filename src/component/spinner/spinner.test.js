import { shallow } from "enzyme";
import React from "react";
import Spinner from "./spinner.component";

describe("Spinner component testing", () => {
  it("Shallow snapshot testing of Spinner component", () => {
    expect(shallow(<Spinner />)).toMatchSnapshot();
  });
});
