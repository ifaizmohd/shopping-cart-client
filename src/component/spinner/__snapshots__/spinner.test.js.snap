// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component testing Shallow snapshot testing of Spinner component 1`] = `ShallowWrapper {}`;
